tests/UI/utils/mailHelper.ts

```diff
@@ -11,31 +11,31 @@ module.exports = {
    * @param config {{smtpPort: string, smtpServer: string, silent:boolean}} Maildev config to start listening
    * @returns {Object}
    */
-  createMailListener(config = global.maildevConfig) {
+  createMailListener(config = global.maildevConfig): MailDev {
     return new MailDev({
       smtp: config.smtpPort,
       silent: config.silent,
     });
   },
   /**
    * Start the maildev listener (listen on 1025 smtp port)
-   * @param mailListener {Object} Maildev server instance
+   * @param mailListener {MailDev} Maildev server instance
    */
-  startListener(mailListener) {
-    mailListener.listen((err) => {
+  startListener(mailListener: MailDev): void {
+    mailListener.listen((err: Error): void => {
       if (err) {
-        throw new Error(err);
+        throw err;
```

Comment:
Nice, this is exactly what we want TypeScript to catch, you had a wrong use before!